index.html

```diff
@@ -1,10 +1,75 @@
 <!DOCTYPE html>
 <html>
     <head>
-    <title></title>
+    <title>TALENT SHOW</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="utf-8"> 
+    <META name="keywords" lang="en-us" content="events,plot,night_life">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     
     </head>
     <body>
+            <div class="topnav" id="myTopnav">
+                    <a href="#home" class="active">Home</a>
+                    <a href="#news">News</a>
+                    <a href="#contact">Contact</a>
+                    <a href="#about">About</a>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```

Comment:
@kyakusahmed , am seeing a jsvascript function "myFuction", wat do u use it for, we are currently doing html and css,